app/Controllers/BaseController.php

```diff
@@ -1,4 +1,8 @@
-<?php namespace App\Controllers;
+<?php
+
+namespace App\Controllers;
```

Comment:
I actually prefer this on the first line, but I'm not going to reject a PR because of it :)